composer.json

```diff
@@ -5,7 +5,7 @@
     "type": "library",
     "license": "MIT",
     "require": {
-        "php": "^7.2",
+        "php": ">7.2.5",
```

Comment:
why `>` rather than `>=` ?

Reply:
I did same as https://github.com/swarrot/SwarrotBundle/blob/master/composer.json#L14.

But I can change to 
```
        "php": "^7.2 || ^8.0",
```

Reply:
I did same as https://github.com/swarrot/SwarrotBundle/blob/master/composer.json#L14

I will change to `>=` to be same as symfony.

Reply:
It fact, as PHP 7.2 is not supported anymore, I'd rather go for `>=7.3`, WDYT? :)